kotlin-analysis-api/src/main/kotlin/com/google/devtools/ksp/impl/FirSealedClassInheritorsProcessorFactoryImpl.kt

```diff
@@ -1,19 +1,11 @@
 package com.google.devtools.ksp.impl
 
 import org.jetbrains.kotlin.analysis.low.level.api.fir.api.services.FirSealedClassInheritorsProcessorFactory
-import org.jetbrains.kotlin.fir.declarations.FirRegularClass
 import org.jetbrains.kotlin.fir.declarations.SealedClassInheritorsProvider
-import org.jetbrains.kotlin.name.ClassId
+import org.jetbrains.kotlin.fir.declarations.SealedClassInheritorsProviderImpl
 
 class FirSealedClassInheritorsProcessorFactoryImpl: FirSealedClassInheritorsProcessorFactory() {
     override fun createSealedClassInheritorsProvider(): SealedClassInheritorsProvider {
-        return SealedClassInheritorProviderImpl()
+        return SealedClassInheritorsProviderImpl
```

Comment:
As mentioned in the internal doc, this one may require upfront FIR facade run. Otherwise, I wonder the attribute about sealed class inheritors are not ready. Well, it'd be just same as `emptyList()` below, for now.

Reply:
It does require a fir analysis, guess we don't really need to rely on this provider to get the sealed inheritors, instead run analyse phase and get the attribute manually should also work?

Reply:
That upstream implementation actually returns the attribute (or empty list if not available), and that attribute is set if fir analysis run up to SEALED_... phase. So I think we need to run FIR analysis, not the whole phases, but up to that phase.

Reply:
But, as per the discussion offline, we can fallback to the original implementation: empty list.